packages/web-ts/src/__test__/CustomForgotPassword.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { render, fireEvent, screen } from "../../test-utils";
+import ForgotPassword from '../components/custom/CustomForgotPassword'
+import { IKUIUserAPI } from '@indykiteone/jarvis-sdk-web'
+
+jest.mock('@indykiteone/jarvis-sdk-web', () => {
+    const original = jest.requireActual('@indykiteone/jarvis-sdk-web');
+    return {
+        ...original,
+        IKUIUserAPI:{
+            sendResetPasswordEmail: jest.fn().mockImplementation(() => Promise.resolve()),
+        }
+    }
+})
+
+describe("All tests for custom ForgotPassword", () => {
```

Comment:
Would it not suffice to write 
```suggestion
describe("custom ForgotPassword", () => {...
```
?
Similarly with all descriptions